src/script/miniscript.cpp

```diff
@@ -11,6 +11,16 @@
 #include <assert.h>
 
```

Comment:
not a big deal, but I think it's more correct to say "CHECKMULTISIG is disabled in Tapscript" in the commit message

Reply:
Done.